modules/apps/site-initializer/site-initializer-liferay-learn/src/main/resources/site-initializer/fragments/group/public-sites-navigation/fragments/public-sites-navigation/index.scss

```diff
@@ -39,26 +39,28 @@ nav.public-sites-navigation {
 			.nav-items-wrapper {
 				background-color: var(--color-brand-primary-darken-5);
 
-				.adt-nav-item {
-					&:hover {
-						.adt-nav-text {
-							background-color: var(--color-action-primary-inverted-hover);
+				@media (min-width: 1200px) {
+					.adt-nav-item {
+						&:hover {
+							.adt-nav-text {
+								background-color: var(--color-action-primary-inverted-hover);
+							}
+
+							.adt-nav-caret-bottom-icon {
+								color: var(--white);
+							}
 						}
 
-						.adt-nav-caret-bottom-icon {
-							color: var(--white);
+						.adt-nav-text {
+							.adt-nav-title {
```

Comment:
If there are no styles on the parent CSS selector (i.e. .adt-nav-text), ideally we would just keep it on the same line like .adt-nav-text .adt-nav-title instead of nesting.